ExperimentConfigSidecar/Services/ConfigService.cs

```diff
@@ -0,0 +1,352 @@
+    /// JSON schema for the pubsub deterioration configuration property.
+    /// </summary>
+    private const string PubsubDeteriorationSchema = """
+    {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "oneOf": [
+            {
+                "type": "object",
+                "properties": {
+                    "delayProbability": {"type": "number"},
+                    "errorProbability": {"type": "number"},
+                    "delay": {"type": "integer"}
+                },
+                "additionalProperties": false
+            },
+            {
+                "type": "null"
+            }
+        ]
+    }
+    """;
+
+    /// <summary>
+    /// JSON schema for the service invocation deterioration configuration property.
+    /// </summary>
+    private const string ServiceInvocationDeteriorationSchema = """
+    {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "$defs": {
+            "item": {
+                "type": "object",
+                "properties": {
+                    "path": {"type": "string"},
+                    "delayProbability": {"type": "number"},
+                    "delay": {"type": "integer"},
+                    "errorProbability": {"type": "number"},
+                    "errorCode": {"type": "integer"}
+                },
+                "additionalProperties": false
+            }
+        },
+        "oneOf": [
+            { "$ref": "#/$defs/item" },
+            {
+                "type": "array",
+                "items": { "$ref": "#/$defs/item" }
+            },
+            {
+                "type": "null"
+            }
+        ]
+    }
+    """;
+
+    /// <summary>
+    /// JSON schema for the artificial memory usage configuration property.
+    /// </summary>
+    private const string MemoryUsageSchema = """
+    {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "oneOf": [
+            {
+                "type": "integer"
+            },
+            {
+                "type": "null"
+            }
+        ]
+    }
+    """;
+
+    /// <summary>
+    /// JSON schema for the artificial CPU usage configuration property.
+    /// </summary>
+    private const string CPUUsageSchema = """
+    {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "$defs": {
+            "item": {
+                "type": "object",
+                "properties": {
+                    "usageDuration": {"type": "integer"},
+                    "pauseDuration": {"type": "integer"}
+                },
+                "required": ["usageDuration", "pauseDuration"],
+                "additionalProperties": false
+            }
+        },
+        "oneOf": [
+            { "$ref": "#/$defs/item" },
+            {
+                "type": "array",
+                "items": { "$ref": "#/$defs/item" }
+            },
+            {
+                "type": "null"
+            }
+        ]
+    }
+    """;
```

Comment:
I would consider moving all Schemas into a sperate file to improve readability

Reply:
extracted with 3a3649e1e0bd2b656e5fd31e50bc84e52c038dc0

Reply:
Feel free to resolve this if it does not work that way in c#:
Usually, if a property is refered to more than once, its good practice to put it in its own variable.
In TS I would do it like this:
`const {ServiceInvocationDeteriorationKey, ... } = ConfigPropertyDefinitions`
And then use the classes (?) directly